api/errors/errors.go

```diff
@@ -298,22 +298,38 @@ func NewBlobstoreUnavailableError(cause error) BlobstoreUnavailableError {
 	}
 }
 
-type CFCreateError struct {
+type ResourceNotFoundError struct {
 	apiError
 }
 
-func NewCFCreateError(cause error) CFCreateError {
-	return CFCreateError{
+func NewResourceNotFoundError(cause error) ResourceNotFoundError {
+	return ResourceNotFoundError{
 		apiError: apiError{
 			cause:      cause,
-			title:      "CF-Create",
+			title:      "ResourceNotFound",
```

Comment:
Is it ok not to prefix these with "CF-"?

Reply:
Ah geez, you're probably right that we want that prefix. I forgot that Cloud Controller prefixes everything with CF- fairly late in the game:

* https://github.com/cloudfoundry/cloud_controller_ng/blob/5614bf7bc92d0e8a91a06fad569b7b738b27e827/app/presenters/v3_error_hasher.rb#L65
* https://github.com/cloudfoundry/cloud_controller_ng/blob/main/app/presenters/v2_error_hasher.rb

Reply:
Updated.